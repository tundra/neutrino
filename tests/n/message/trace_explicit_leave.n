# Copyright 2014 the Neutrino authors (see AUTHORS).
# Licensed under the Apache License, Version 2.0 (see LICENSE).

def $really_fire_abort() => leave.get_me_outta_here(10, "foo");
def $fire_abort() => $really_fire_abort();
do $fire_abort();
