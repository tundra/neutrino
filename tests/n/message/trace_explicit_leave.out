--- backtrace ---
- leave.get_me_outta_here(10, "foo")
- :really_fire_abort()
- :fire_abort()
Error: %<condition: Signal(escape)>
